doc/design.md

```diff
@@ -0,0 +1,31 @@
+# Design
+
+## Introduction
+The Transformers Extension proposes to use large pre-trained models using 
+[Transformers API](https://huggingface.co/docs/transformers/index) in Exasol 
+Database. These models can be used for many predictive tasks such as text 
+classification, question-answering, text generation  etc. in the field of NLP.
```

Comment:
```suggestion
The Transformers Extension proposes to use large pre-trained models together with Exasol Database's [Transformers API](https://huggingface.co/docs/transformers/index). These models can be used for many predictive tasks such as text 
classification, question-answering, text generation etc. in the field of NLP.
```

Reply:
Transformers API is not the one we provide. This is the one we use.